src/redux/store.js

```diff
@@ -0,0 +1,15 @@
+import { configureStore } from '@reduxjs/toolkit';
+// import { createLogger } from 'redux-logger';
+import greetReducer from './greet/greetSlice';
+
+// const logger = createLogger();
+
+const store = configureStore({
+  reducer: {
+    greet: greetReducer,
+  },
+//   ,
+//   middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(logger),
+});
```

Comment:
- [ ] Kindly remove all unused commented-out codes. Leaving such codes in your code base makes your work look untidy and hard to read. Kindly remove these commented-out codes. 👍